src/runtime-rs/crates/shim/src/args.rs

```diff
@@ -117,20 +117,19 @@ mod tests {
             result: Result<()>,
         }
 
-        let default_id = "1dfc0567".to_string();
-        let default_namespace = "ns1".to_string();
+        let default_id = "default_id".to_string();
```

Comment:
im not seeing the reason of this modification, but it is fine :)

Reply:
When I first saw the code here, I thought the id 1dfc0567 has a special meaning. But it's not indeed. So I want to use a meaningful id to help the later readers.